### PR TITLE
Osm upload

### PR DESCRIPTION
**Pull request by mattwigway**

### CODE REVIEW

**Comment by mattwigway** [comment]
@trevorgerhardt do we know why the uuid of the mock project changed?

**Comment by mattwigway** [comment]
So it appears that it is not possible to make the tests pass because the uuid of that mock project changes every time you run the test suite.

**Comment by mattwigway** [comment]
@trevorgerhardt tests are fixed! Problem was that because I was stringifying the project by hand now the whole stringified JSON was winding up in the snapshot instead of [object Object] or whatever.

File: __mocks__/uuid.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const uuid = require.requireActual('uuid')
+
+module.exports = {
+  ...uuid,
+  v4: () => '00000000-0000-0000-0000-000000000000'
```
Comment (trevorgerhardt) [inline __mocks__/uuid.js]: 👌 👌

**Comment by trevorgerhardt** [comment]
I like that implementation! We've worked around the `uuid`s in the past—no need to anymore!

**Pull request merged**